assets/js/modules/analytics-4/datastore/properties.test.js

```diff
@@ -1130,6 +1159,87 @@ describe( 'modules/analytics-4 properties', () => {
 						.hasMismatchedGoogleTagID()
 				).toBe( true );
 			} );
+
+			it( 'should set Google Tag container destination IDs in module setting', async () => {
+				provideUserAuthentication( registry, {
+					grantedScopes: [ TAGMANAGER_READ_SCOPE ],
+				} );
+
+				provideModules( registry, [
+					{
+						slug: 'analytics-4',
+						active: true,
+						connected: true,
+					},
+				] );
+
+				const measurementID = 'G-2B7M8YQ1K6';
+				const containerMock = fixtures.container[ measurementID ];
+
+				const {
+					googleTagID,
+					googleTagAccountID,
+					googleTagContainerID,
+				} = fixtures.googleTagSettings;
+
+				registry.dispatch( MODULES_ANALYTICS_4 ).receiveGetSettings( {
+					measurementID,
+					googleTagID,
+					googleTagAccountID,
+					googleTagContainerID,
+					googleTagLastSyncedAtMs: 1670123456789,
+				} );
+
+				fetchMock.getOnce( containerLookupEndpoint, {
+					body: containerMock,
+					status: 200,
+				} );
+
+				fetchMock.getOnce( containerDestinationsEndpoint, {
+					body: containerDestinationsMock,
+					status: 200,
+				} );
+
+				const ga4Settings = {
+					measurementID,
+					googleTagAccountID,
+					googleTagContainerID,
+					googleTagID,
+				};
+
+				fetchMock.postOnce( ga4SettingsEndpoint, {
+					body: {
+						...ga4Settings,
+						googleTagLastSyncedAtMs: Date.now(),
```

Comment:
This is a minor niggle, but `Date.now()` doesn't _precisely_ reflect the value that this endpoint will return, as it will actually be a few milliseconds later.

This could be slightly confusing when reading the test as the inclusion of this value could suggest it's actually going to be used for something and is entirely accurate.

I'd suggest adding a brief comment to explain it's just there for illustrative purposes, reflecting how the value will be calculated at the point of dispatch.